lib/CIGateway.py

```diff
@@ -1,15 +1,15 @@
-import io
-from subprocess import Popen, call
-from jujubigdata import utils
+from io import open
```

Comment:
This seems unnecessary.  I vote for dropping it, unless I'm missing some reason to use the `io.open` alias instead of the normal built-in.